Initial Slack Stats Bot Design

kenrosefire opened the pull request:
Information about design is found in file header. It is not available for public distribution on Slack yet due to a missing Flask Server. However, it can be used internally with the Sockets API and a user-created bot. A link to the project online can be found here: https://replit.com/@kenrosefire/Slack-Bot#main.py

kenrosefire committed:
SlackStatsBot

kenrosefire committed:
Empty DB

kenrosefire committed:
Update main.py

twood02 commented:
This sounds great! Want to connect it to my research group's Slack and show me how it works sometime? I sent you an invite. Message me when you join and we can schedule a time to try it out.

kenrosefire committed:
Update main.py

kenrosefire committed:
Added oauth and fixed date issues

App is ready for public installation. Users can install by visiting https://Slack-Bot.kenrosefire.repl.co.

kenrosefire committed:
Updated documentation

kenrosefire commented:
The date issues have been fixed and public distribution has been enabled. Error logging has also been added. The bot can be installed at this link: https://Slack-Bot.kenrosefire.repl.co.

kenrosefire committed:
Create stats.py

kenrosefire committed:
Create Main.sublime-menu

kenrosefire committed:
Create Default.sublime-commands

kenrosefire commented:
I created a new sublime text program for statistics logging. It has the following limitations:
- No long term storage
- Sublime text has no way for developers to modify the bottom bar like in the original Atom code. In my program, a button must be pressed to trigger a popup.
- Not ready for public distribution yet (To be fixed)
